proxy/director.go

```diff
@@ -126,7 +126,7 @@ func timedUnavailabilityFunc(wait time.Duration) func(*endpoint) {
 	return func(ep *endpoint) {
 		time.AfterFunc(wait, func() {
 			ep.Available = true
-			log.Printf("proxy: marked endpoint %s available", ep.URL.String())
+			log.Printf("proxy: marked endpoint %s available to retry it until refresh interval time arrived.", ep.URL.String())
```

Comment:
1. it fails to get response from the endpoint
2. it marks endpoint as unavailable
3. it will not try the endpoint in refresh interval
4. it marks endpoint as available to hope it back online

So it will try the endpoint after the refresh interval